apps/server/src/schema/organization/org-types.ts

```diff
@@ -86,3 +91,12 @@ export const inviteLinkDto = builder.simpleObject('InviteLinks', {
     role: t.field({ type: OrganizationRoleEnumDto, nullable: false }),
   }),
 });
+
+export const tierEnum = builder.enumType('tier', {
```

Comment:
```suggestion
export const TierEnum = builder.enumType('tier', {
```
and let's capitalize `inviteLinkDto` as well please